zerver/tests/test_openapi.py

```diff
@@ -819,6 +819,7 @@ def test_generate_and_render_curl_with_array_example(self) -> None:
             "curl -sSX GET -G http://localhost:9991/api/v1/messages \\",
             "    -u BOT_EMAIL_ADDRESS:BOT_API_KEY \\",
             "    --data-urlencode anchor=43 \\",
+            "    --data-urlencode include_anchor=false \\",
```

Comment:
I think we don't want this line in the example; these are not intended to show all options and also isn't `include_anchor=false`  an error with the other parameters here?

Reply:
This doesn’t show up in the actual documentation (due to `x-curl-examples-parameters`), only this test.

Reply:
OK.